api/src/main/scala/hmda/api/model/InstitutionWrapper.scala

```diff
@@ -3,7 +3,7 @@ package hmda.api.model
 import hmda.model.institution.InstitutionStatus
 
 case class InstitutionWrapper(
-  id: Int,
+  id: String,
```

Comment:
Sort of. This object is a wrapper around what really gets persisted (so that we don't send out all details about an institution, that are not needed at the API level), see the the lines above where id.id.toString is called. My concern is that the internal [Institution](https://github.com/cfpb/hmda-platform/blob/master/model/src/main/scala/hmda/model/institution/Institution.scala#L7) object doesn't model its id as a String. I'm ok with merging this one and ticketing this as a different issue.